### ISSUE TITLE
stabilize the block generation time at about 15 seconds in a smooth way

### ISSUE THREAD

**Issue by vang1ong7ang** [bane-labs/go-ethereum#151]
like the issue discussed in https://github.com/neo-project/neo/issues/2918

stabler block generation time is beneficial to the design and development of on-chain applications

**Comment by dusmart** [comment]
Yeah. Seems that Roman had a similar opinion on a specific point that will extending the block generation time. https://github.com/bane-labs/go-ethereum/issues/20
